src/features/dynamicmenu/DynamicMenu.gd

```diff
@@ -0,0 +1,43 @@
+extends Control
+
+func reveal_slide_child(node: Control) -> void:
```

Comment:
I see a lot of shared code in these two functions. Namely around the spacer and its behavior. Can we combine some of this shared logic?